Source/Core/Core/ConfigManager.cpp

```diff
@@ -644,6 +644,7 @@ void SConfig::SetRunningGameMetadata(const IOS::ES::TMDReader& tmd, DiscIO::Plat
 
 void SConfig::SetRunningGameMetadata(const std::string& game_id)
 {
+  SConfig::GetInstance().m_details_game_id = game_id;
```

Comment:
Couple questions:
- why SConfig::GetInstance() here? isn't this in a member function for the instance itself?
- I'm looking at SetRunningGameMetadata below, and it looks like there is a m_game_id member already. Is m_details_game_id needed as well?

Reply:
Fixed comment 1, but the reasoning behind the need for m_details_game_id is that m_game_id gets changed later in boot, so to preserve the game ID we need it needs to be stored elsewhere.

Reply:
ok. I think it would be easier to read if we moved the `m_details_game_id = game_id;` line to the `SetRunningGameMetadata(const std::string& game_id, const std::string& gametdb_id, u64 title_id, u16 revision, DiscIO::Region region)` 
function.

Where the m_game_id, m_gametdb_id, etc. is being set